app/pages/sections/order-items/catalogue-solutions/order-item/flat/ondemand/manifest.json

```diff
@@ -12,7 +13,7 @@
     },
     "quantity": {
       "id": "quantity",
-      "mainAdvice": "How many do you think you'll need?",
+      "mainAdvice": "How many do you think you’ll need?",
```

Comment:
I don't think we need to change the apostrophe character.

Reply:
reverted to original